src/mmds/src/token.rs

```diff
@@ -219,7 +234,11 @@ impl TokenAuthority {
             .map_err(Error::EntropyPool)?;
 
         // Create cipher entity to handle encryption/decryption.
-        Ok(Aes256Gcm::new(Key::from_slice(&key)))
+        let cipher = Aes256Gcm::new(Key::from_slice(&key));
+        // Free contents of key.
+        key.zeroize();
```

Comment:
I'd argue we don't need to use this since Rust variables are always initialized. Furthermore, this memory belongs to the host and it is inaccessible to the guest. This means we would also get rid of the dependency mentioned earlier.

Reply:
Resolving because of https://github.com/firecracker-microvm/firecracker/pull/2835#discussion_r781969770